cinema/views.py

```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
 )
```

Comment:
Code Style: Please group your imports with parentheses consistently. You have a trailing comma after `OrderListSerializer` on line 26 and then closing parenthesis on a new line (line 27). Here is a good example: 

python
from cinema.serializers import (
    GenreSerializer,
    ActorSerializer,
    ...
    OrderListSerializer
)
```